internal/query/current_sequence.go

```diff
@@ -0,0 +1,72 @@
+package query
+
+import (
+	"context"
+	"database/sql"
+	errs "errors"
+	"time"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/caos/zitadel/internal/errors"
+)
+
+type LatestSequence struct {
+	Sequence  uint64
+	Timestamp time.Time
+}
+
+func prepareLatestSequence() (sq.SelectBuilder, func(*sql.Row) (*LatestSequence, error)) {
+	return sq.Select(
+			CurrentSequenceColCurrentSequence.identifier(),
+			CurrentSequenceColTimestamp.identifier()).
+			From(currentSequencesTable.identifier()).PlaceholderFormat(sq.Dollar),
+		func(row *sql.Row) (*LatestSequence, error) {
+			seq := new(LatestSequence)
+			err := row.Scan(
+				&seq.Sequence,
+				&seq.Timestamp,
+			)
+			if err != nil {
+				if errs.Is(err, sql.ErrNoRows) {
+					return nil, errors.ThrowNotFound(err, "QUERY-gmd9o", "errors.current_sequence.not_found")
+				}
+				return nil, errors.ThrowInternal(err, "QUERY-aAZ1D", "errors.internal")
```

Comment:
error message translations are missing, i will not comment on each error message, please have a look at all of them